src/lint.ts

```diff
@@ -186,15 +186,20 @@ function range(minVersion: TsVersion, maxVersion: TsVersion): ReadonlyArray<TsVe
         assert(maxVersion === "local");
         return ["local"];
     }
```

Comment:
this function is the only substantive change, from `supported` to `shipped`+`latest`.